### PR TITLE
sonarqube 6.2

### PR DESCRIPTION
**Pull request by igorbolic**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/sonarqube.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sonarqube < Formula
   desc "Manage code quality"
   homepage "http://www.sonarqube.org/"
```
Comment (fxcoudert) [inline Formula/sonarqube.rb]: Audit says: `The URL http://www.sonarqube.org/ could use HTTPS rather than HTTP`

File: Formula/sonarqube.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-  url "https://sonarsource.bintray.com/Distribution/sonarqube/sonarqube-6.1.zip"
-  sha256 "21ecd2a1c85bfb438411e44d7b9edcca310e8d6fca96b6da97efe9481ff3b806"
+  url "https://sonarsource.bintray.com/Distribution/sonarqube/sonarqube-6.2.zip"
+  sha256 "6bb187b8568d493131952e3b66e65940992f1a62c7962c1bf5801f6f48f1c160"
 
   bottle :unneeded
```
Comment (fxcoudert) [inline Formula/sonarqube.rb]: The `aix`, `hpux`, and `solaris` can be removed: they are not shipped anymore.

`macosx-universal-32` is not shipped anymore either, so:
- remove the block including `rm_rf "bin/macosx-universal-*"`
- remove the lines around `bin.install_symlink "#{libexec}/bin/macosx-universal-64/sonar.sh" => "sonar"`

Also, remove the line `rm_f Dir["war/*.bat"]` as those do not exist anymore.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/sonarqube.rb (hunk 2 above)
Comment (igorbolic) [inline Formula/sonarqube.rb]: @fxcoudert I've pushed the code changes

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @igorbolic for the pull request!

**Pull request merged**